### DISCUSSION TITLE
No "with-Java" for Freeplane 1.12.8?

### DISCUSSION THREAD

**Discussion by jockster74** [freeplane/freeplane#2143] [General]
Hi. 
  I got a notification about the new version of Freeplane 1.12.8. When I went to the downloads page in SourceForge, there is no "Freeplane-Setup-with-Java-1.12.8.exe" (like there is for 1.12.6, for example. How come?

**Comment by HaaiHenkie** [comment]
See #2113, for Windows there is now only a download with java and it is not in the name anymore.
